plasmapy/utils/pytest_helpers/pytest_helpers.py

```diff
@@ -18,16 +18,16 @@
 
 from typing import Any, Callable, Dict
 
-from plasmapy.utils.error_messages import _exc_str, _represent_result, call_string
-from plasmapy.utils.exceptions import PlasmaPyWarning
-from plasmapy.utils.pytest_helpers.exceptions import (
+from plasmapy.tests.helpers.exceptions import (
```

Comment:
Note: This import will cause `plasmapy.tests` to be automatically import when `plasmapy` is imported.  This will stop happening then your full architecture is implemented, assuming `plasmapy.tests.helpers` are only used within test directories.  (**Probably good to make a note about this on the "Testing Guidelines" page.**)

Reply:
Good point!  I hadn't realized that, though it's good that this problem will go away once the whole refactoring is done.